File: GestaoFilmesAPI/GestaoFilmesAPI.Data/Models/Filme.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GestaoFilmesAPI.Data.Models
{
    public class Filme
    {
        public int Id { get; set; }
        public string Titulo { get; set; }
        public string Diretor { get; set; }
        public int GeneroId { get; set; }
        public string Sinopse { get; set; }
        public int? Ano { get; set; }

        public Genero Genero { get; set; }
    }
}

File: GestaoFilmesWEB/src/app/models/Filme.ts
import { Genero } from './genero';

export class Filme {

    constructor() {
        this.id = null;
        this.titulo = null;
        this.diretor = null
        this.generoId = null;
        this.sinopse = null
        this.ano = null;
        this.genero = null;
    }

    id: number;
    titulo: string;
    diretor: string;
    generoId: number;
    sinopse: string;
    ano: number;

    genero: Genero;
}
